README.md

```diff
@@ -2,70 +2,98 @@
    </a>
-
 </div>
 
+<h1 align="center">
+   <p>Open Source Unstructured API</p>
+</h1>
 
----
-
-<h3 align="center">
-  <p>General Pre-Processing Pipeline for Documents</p>
-</h3>
-
-This repo implements a pre-processing pipeline for the following documents. Currently, the pipeline is capable of recognizing the file type and choosing the relevant partition function to process the file.
+This repo implements a FastAPI server with the partitioning functionality of the [Unstructured library](https://github.com/Unstructured-IO/unstructured). It has one endpoint that accepts any of the following filetypes, and returns the contents as structured JSON.
```

Comment:
If we don't want to highlight the open source library as much, consider not linking to it from here.

Reply:
Also, you can target more than one endpoint, right?